src/xcode/ENA/ENA/Source/Client/HTTP Client/__tests__/HTTPClient+AuthorizationOTPElsTests.swift

```diff
@@ -406,4 +406,50 @@ final class HTTPClientAuthorizationOTPElsTests: CWATestCase {
 		XCTAssertNotNil(expectedOtpError)
 		XCTAssertEqual(expectedOtpError, .invalidResponseError)
 	}
+	
+	func testGIVEN_AuthorizeOTP_WHEN_NoNetworkConnection_THEN_NetworkErrorReturned() {
+		// GIVEN
+		let url = URL(staticString: "https://localhost:8080")
+		let notConnectedError = NSError(
+			domain: NSURLErrorDomain,
+			code: NSURLErrorNotConnectedToInternet,
+			userInfo: nil
+		)
+
+		let session = MockUrlSession(
+			data: nil,
+			nextResponse: HTTPURLResponse(
+				url: url,
+				statusCode: 500,
+				httpVersion: nil,
+				headerFields: nil
+			),
+			error: notConnectedError
+		)
+
+		let stack = MockNetworkStack(
+			mockSession: session
+		)
+
+		let expectation = self.expectation(description: "completion handler is called without an error")
+		let otp = "OTPFake"
+		let ppacToken = PPACToken(apiToken: "APITokenFake", deviceToken: "DeviceTokenFake")
+
+		// WHEN
+		var expectedOtpError: OTPError?
+		HTTPClient.makeWith(mock: stack).authorize(otpEls: otp, ppacToken: ppacToken, completion: { result in
+			switch result {
+			case .success:
+				XCTFail("success should not be called")
+			case .failure(let otpError):
+				expectedOtpError = otpError
```

Comment:
Why no check in place:
```suggestion
				XCTAssertEqual(otpError, .noNetworkConnection)
```